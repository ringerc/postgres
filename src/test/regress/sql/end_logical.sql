\copy (SELECT data FROM start_logical_replication('regression_slot', 'now', 'include-xids', '0')) TO '/tmp/testresult';
SELECT stop_logical_replication('regression_slot');
