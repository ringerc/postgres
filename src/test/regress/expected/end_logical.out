\copy (SELECT data FROM start_logical_replication('regression_slot', 'now', 'include-xids', '0')) TO '/tmp/testresult';
SELECT stop_logical_replication('regression_slot');
 stop_logical_replication 
--------------------------
                        0
(1 row)

